packages/shared/web/src/header/mobile-navigation/mobile-navigation.tsx

```diff
@@ -129,90 +161,64 @@ export function MobileNavigation({
-          {navLinks.map((t) => (
-            <Tab
-              key={t.key}
-              icon={t.iconImg}
-              iconPosition="start"
-              label={t.label}
-              href={t.link}
-              value={t.link}
-              rel={t.external && t.target === '_blank' ? 'noreferrer' : ''}
-              target={t.target || '_self'}
-              component="a"
-              onClick={handleCloseNavMenu}
-              sx={{
-                display: {
-                  xs: 'flex',
-                  md: 'none',
-                },
-                '.MuiSvgIcon-root': {
-                  color: theme.palette.common.black,
-                },
-                '&.MuiTab-root, .MuiTab-labelIcon': {
-                  opacity: 1,
-                  color: theme.palette.common.black,
-                  textTransform: 'capitalize',
-                  fontSize: '1rem',
-                  justifyContent: 'flex-start',
-                  maxWidth: 'none',
-                  width: '90%',
-                  padding: '0px',
-                  margin: 'auto',
-                  borderBottom:
-                    t.key !== 'provide' ? '1px solid #089CA3' : 'none',
-                },
-                '&:hover': {
-                  svg: {
-                    filter:
-                      'invert(63%) sepia(16%) saturate(1939%) hue-rotate(134deg) brightness(91%) contrast(96%)',
+          <Box sx={{ boxShadow: theme.shape.shadowStandard }}>
+            {navLinks.map((t) => (
+              <Tab
+                key={t.key}
+                icon={t.iconImg}
+                iconPosition="start"
+                label={t.label}
+                href={t.link}
+                value={t.link}
+                rel={t.external && t.target === '_blank' ? 'noreferrer' : ''}
+                target={t.target || '_self'}
+                component="a"
+                sx={{
+                  display: {
+                    xs: 'flex',
+                    md: 'none',
                   },
-                  color: theme.palette.primary.light,
-                },
-                '&.Mui-selected': {
-                  fontWeight: 700,
-                },
-              }}
-            />
-          ))}
-          <MobileSubNav
-            mobileSubNavLinks={mobileSubNavLinks}
-            handleSideDrawer={handleSideDrawer}
-          />
+                  '.MuiSvgIcon-root': {
+                    color: theme.palette.common.black,
+                  },
+                  '&.MuiTab-root, .MuiTab-labelIcon': {
+                    opacity: 1,
+                    color: theme.palette.common.black,
+                    textTransform: 'capitalize',
+                    fontSize: '1rem',
+                    justifyContent: 'flex-start',
+                    maxWidth: 'none',
+                    width: '90%',
+                    padding: '0px',
+                    margin: 'auto',
+                    borderBottom: '1px solid #089CA3',
```

Comment:
Is this border color in the palette somewhere?

Reply:
This color does not exist in our theme. Replaced this with the standard border